backend/entities/settings/api.js

```diff
@@ -0,0 +1,47 @@
+const getSettings = require('./controller').getSettings;
+const updateBoardName = require('./controller').updateBoardName;
+const updateBoardLogo = require('./controller').updateBoardLogo;
+
+/**
+ * settings apis
+ * @param  {Object} app
+ */
+const settingsAPI = (app) => {
+  // get admin settings
+  app.get('/api/settings/all_settings', (req, res) => {
+    if (req.user && req.user.role === 'admin') {
+      getSettings().then(
+        (data) => {
+          res.send(data);
+        },
+        (error) => {
+          res.send(error);
+        }
+      );
+    } else res.send({error: 'You are not admin buddy 😛'});
+  });
+
+  // update board name
+  app.put('/api/settings/name', (req, res) => {
```

Comment:
Do we want to make it so that we have 1 settings update endpoint vs separate endpoints for child values?

Reply:
Yeah that might make more sense with the current UX (form and submit button), but if you add in things like a theme toggle, or the forum locking dropdowns, it might make sense to have separate endpoints.

Reply:
I can definitely change it to be the former for now though!

Reply:
I've decided to leave them as separate children for now, as it will be helpful to keep them that way for when I add the whitelisting and forum locking